typescript/phoenix-sdk/src/market.ts

```diff
@@ -565,9 +565,14 @@ export class Market {
 
   getPriceDecimalPlaces(): number {
     let target =
-      Math.pow(10, this.data.header.quoteParams.decimals) /
+      (Math.pow(10, this.data.header.quoteParams.decimals) *
+        this.data.header.rawBaseUnitsPerBaseUnit) /
       toNum(this.data.header.tickSizeInQuoteAtomsPerBaseUnit);
 
+    if (target === 0) {
+      return 0;
```

Comment:
this should definitely never be zero; if it is we should throw

Reply:
i guess in JS it won't be. I think <= is the better check here